cupyx/scipy/sparse/csr.py

```diff
@@ -5,12 +5,16 @@
     _scipy_available = False
 
 import cupy
+from cupy.core import _backend
 from cupy import cusparse
 from cupyx.scipy.sparse import base
 from cupyx.scipy.sparse import compressed
 from cupyx.scipy.sparse import csc
-if cupy.cuda.cub_enabled:
+
+try:
     from cupy.cuda.cub import device_csrmv
```

Comment:
Actually this violates the dev guideline...Let's fix it properly:
```suggestion
    from cupy.cuda import cub
```